src/components/Common/LanguagePicker.js

```diff
@@ -239,9 +239,9 @@ export const LanguagePicker = ({
         )}
       />
 
-      {/* eslint-disable indent */
-      languageCode !== 'en-US' &&
-        languageCode !== 'ja-JP' && (
+      {
+        /* eslint-disable indent */
```

Comment:
I suspect this comment can be removed now, since prettier was added in eslint rules, it shouldn't conflict. There are a bunch of other cases

Reply:
done